src/entity-manager/EntityManagerFactory.ts

```diff
@@ -16,6 +18,9 @@ export class EntityManagerFactory {
         if (connection.driver instanceof MongoDriver)
             return new MongoEntityManager(connection);
 
+        // if (connection.driver instanceof SqljsDriver)
+        //     return new SqljsEntityManager(connection, queryRunner);
```

Comment:
I had to remove these lines becaus otherwise I get a `TypeError: Object prototype may only be an Object or null: undefined` error. Which, as far as I can tell, is because `require('./EntityManager')` inside `SqljsEntityManager` returns `{}`. But I don't really know why, the only information I found online was, that it could be a problem with circular dependency. But `MongoEntityManager` looks the same and works without problems